addon/content/reducer.js

```diff
@@ -107,7 +128,167 @@ const attachmentActions = {
   },
 };
 
+// TODO: Once the WebExtension parts work themselves out a bit more,
+// determine if this is worth sharing via a shared module with the background
+// scripts, or if it doesn't need it.
+const kScrollUnreadOrLast = 0;
+const kScrollSelected = 1;
+
+async function setupConversationInTab(params, isInTab) {
+  let scrollMode = params.get("scrollMode");
+  if (scrollMode) {
+    scrollMode = parseInt(scrollMode);
+  } else {
+    scrollMode = kScrollUnreadOrLast;
+  }
+  // If we start up Thunderbird with a saved conversation tab, then we
+  // have no selected message. Fallback to the usual mode.
+  if (
+    scrollMode == kScrollSelected &&
+    !topMail3Pane(window).gFolderDisplay.selectedMessage
+  ) {
+    scrollMode = kScrollUnreadOrLast;
+  }
+
+  if (window.frameElement) {
+    window.frameElement.setAttribute("tooltip", "aHTMLTooltip");
+  }
+  // let willExpand = parseInt(params.get("willExpand"));
```

Comment:
Spurious comment?

Reply:
This actually ties up with code further down that's commented out. I'll move it closer.